package.json

```diff
@@ -31,7 +31,7 @@
         "url": "https://github.com/DigiByte-Core/digihash.git"
     },
     "dependencies": {
-        "stratum-pool": "https://github.com/DigiByte-Core/node-stratum-pool.git",
```

Comment:
Eventually, I think it would be good to update this repo with your local changes and revert this to the DigiByte-Core location.